.github/workflows/lint.yml

```
@@ -45,6 +45,6 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
-      - uses: DavidAnson/markdownlint-cli2-action@b4c9feab76d8025d1e83c653fa3990936df0e6c8 # v16
+      - uses: DavidAnson/markdownlint-cli2-action@db43aef879112c3119a410d69f66701e0d530809 # v16
```

I think I should pin this with nixpkgs version. They does not provide version specifier https://github.com/DavidAnson/markdownlint-cli2-action/issues/94

---

And this comment is not updated...

https://github.com/DavidAnson/markdownlint-cli2-action/tags

```suggestion
      - uses: DavidAnson/markdownlint-cli2-action@db43aef879112c3119a410d69f66701e0d530809 # v17
```

dependabot bug?

---

As I did in https://github.com/pankona/pankona.github.com/pull/314#discussion_r1759336084

---

Removed in https://github.com/kachick/dotfiles/commit/31727e19851188eb8ae0839f1918712d838bb9e4